### PR TITLE
add new version kafka and apiversions support

### PR DESCRIPTION
**Pull request by hanbing0715**

### CODE REVIEW

File: broker.go [hunk 1]
```diff
@@ -355,6 +355,17 @@ func (b *Broker) DescribeGroups(request *DescribeGroupsRequest) (*DescribeGroups
...
+	response := new(ApiVersionsResponse)
+
+	err := b.sendAndReceive(request, response)
+	if err != nil {
+		return nil, err
+	}
+
+	return response, nil
+}
+
```
Comment (wvanbergen) [inline broker.go]: Can you add a test for this new API?

Comment (hanbing0715) [inline broker.go]: i have already add this test in broker_test.go

**Comment by wvanbergen** [comment]
One final thing: can you rebase your branch and get squash the last 4 commits into one?

**Comment by hanbing0715** [comment]
i have do the rebase and squash

**Comment by wvanbergen** [comment]
@eapache final :eyes: before merging this?

**Pull request merged**